<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QMainWindow" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>248</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="keyLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Key:</string>
     </property>
    </widget>
    <widget class="QLabel" name="apiURLLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Api URL:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="apiURLLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>521</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="keyLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>521</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="iconURLLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Icon URL:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="iconURLLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>521</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
